apps/site/lib/site_web/templates/event/_add_button.html.eex

```diff
@@ -1,5 +1,5 @@
 <div class="m-event__status-message <%= @class %>">
-  <%= if !event_ended(@range) do %>
+  <%= if event_status(@range) != :ended do %>
```

Comment:
Do we want to show the link when the event is `:started` as well?

Reply:
Ah, hm... The mock didn't include an "in progress" state.  Just button or "ended".  I'll ask Yinka